parlai/crowdsourcing/tasks/fast_acute/analysis.py

```diff
@@ -46,17 +48,18 @@
 AGREEMENT_TIES_OKAY = False
 # NOTE: these could be added as flags if desired
 
-# Prepended to checkbox columns in self.dataframe
-CHECKBOX_PREFIX = 'checkbox: '
-
 
 def setup_args():
     """
     Setup appropriate args.
     """
     parser = ParlaiParser(False, False)
     parser.add_argument(
-        '-id', '--run-id', type=str, default=None, help='run id to analyze'
+        '-id',
```

Comment:
nit: `'-ids', '--run-ids'` since this can now be more than one?

Reply:
Makes sense - changing this